scripts/packaging/generic_distro.py

```diff
@@ -54,12 +54,11 @@ def get_webots_version():
     # get package version
     try:
         with open(os.path.join(os.getenv('WEBOTS_HOME'), 'scripts', 'packaging', 'webots_version.txt'), 'r') as version_file:
-            version = version_file.readline()
-            match = re.search(r'^(.{6,15})( .* (\d+))?', version)
-            global package_version
-            if match.group(2):
-                return match.group(0) + '-rev' + match.group(2)
-            return match.group(0)
+            version = version_file.readline().strip()
+            parts = version.split(' ')
+            if parts[2]:
```

Comment:
This will throw errors:
```suggestion
            if len(parts) > 2:
```